portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,135 @@
+    <h3>08/18-05/19</h3>
+    <ul>
+      <li>Used the ServiceNow support ticket system to help customers.</li>
+      <li>Moved, configured, wiped, cleaned, and imaged computers.</li>
+      <li>Worked at a help desk, troubleshooting IT issues in person and over the phone.</li>
+    </ul>
+  </div>
+  <div class="section" id="projects">
+    <h1>Projects</h1>
+    <h2>Fare-Comparison</h2>
+    <p>
+      Fare Comparison is an AppEngine application that I contributed to while working in a team of three developers during
+      Google’s 2018 Computer Science Summer Institute in Atlanta. It served as a way to quickly and easily compare Lyft and
+      Uber prices for a given ride. It utilized Google APIs to implement location autocomplete, and to return a map of the
+      fastest route between the inputted source and destination locations. Python, HTML, CSS, and Javascript were all integral
+      tools.
+    </p>
+    <p>
+      Unfortunately, as a result of a change in Lyft and Uber’s API policies, Fare Comparison no longer functions as intended.
+    </p>
+    <h1>Coming Soon</h1>
+    <h2>Attention</h2>
+    <p>
+      A group of Go command line tools designed to help users with ADHD to measure and manage their symptoms. This project
+      will be heavily inspired by peer-reviewed literature. It will include tools for measuring distractibility, maintaining
+      focus on a particular task, and ordering tasks by priority.
+    </p>
+    <p>
+      The name is subject to change to become more interesting.
+    </p>
+  </div>
+  <div class="section" id="fun-stuff">
+    <h1>Fun Stuff</h1>
+    <p>
+      You made it all the way to the bottom of my portfolio! Stay skeptical, sometimes the fun things in life require a bit of
+      digging :). Here are some things I probably wouldn’t include on a regular resume!
+      <ul>
+        <li>At my Google internship in Chicago, I learned how to juggle</li>
+        <li>I enjoy playing DnD, particularly homebrew versions</li>
+        <li>I have a significant interest in personal finance (budgeting, credit, retirement) and love to share what I know with the people around me when they ask</li>
+        <li>My mom is Jamaican, so I can understand a bit of our national language, Patois</li>
+        <li>I learned a lot of the sounds of Spanish growing up in North Miami Beach, but I couldn’t roll my r’s until I was 18! It took months of daily practice</li>
+        <li>I’ve been doing problems on Project Euler since I was 14, but I only completed my first problem with a difficulty higher than 5% at 19! To be fair, I wasn’t particularly dedicated</li>
+        <li>He tomado cinco clases de español en mi universidad. Mi español no está en el nivel de un nativo, ¡pero es bastante bueno para mi!</li>
+        <li>I earned the small scar I have on my face in a battle over a blankie I had when I was 5</li>
+      </ul>
+    </p>
+  </div>
+</div>
+</body>
```

Comment:
Prefer using classes over ids unless you actually need it to be id for something such for document.getElementById or for linking to a section of a page.

Reply:
I realize that this isn't an amazing reason, but I made many of those IDs because I wanted to make a table of contents. It ended up being outside the scope of the MVP.